Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -755,6 +755,44 @@ void wpe_view_unmap(WPEView* view)
     g_object_notify_by_pspec(G_OBJECT(view), sObjProperties[PROP_MAPPED]);
 }
 
+/**
+ * wpe_view_lock_pointer:
+ * @view: a #WPEView
+ *
+ * Disable the movements of the pointer in @view, locking it to one position.
```

Comment:
This comment is confusing, we are not locking the pointer to one position, but to a particular area. I would explain here that while the pointer is locked, mouse move events are relative instead of absolute.